src/shared/components/oncoprint/OncoprintUtils.ts

```diff
@@ -899,6 +903,8 @@ export function makeClinicalTracksMobxPromise(
                     }
                 } else if (attribute.datatype === 'STRING') {
                     ret.datatype = 'string';
+                    (ret as any).category_to_color =
```

Comment:
how come any?

Reply:
typescript doesnt handle this well, union type